README.md

```diff
@@ -1,95 +1,105 @@
-# puppetlabs-distelli_agent
-
-#### Table of Contents
-
-
-1. [Module Description - What the module does and why it is useful](#module-description)
-1. [Setup - The basics of getting started with the Distelli agent](#setup)
-1. [Usage - Configuration options and additional functionality](#usage)
-1. [Reference - An under-the-hood peek at what the module is doing and how](#reference)
-1. [Limitations - OS compatibility, etc.](#limitations)
-1. [Development - Guide for contributing to the module](#development)
+# puppetlabs-pipelines: Configure the Puppet Pipelines agent
 
+The Pipelines module installs, configures, and manages the agent for [Puppet
+Pipelines](https://puppet.com/products/puppet-pipelines-applications) on
+Linux, macOS, and Windows. It can be used on build, application, and key
+management servers.
 
-## Module description
+#### Table of Contents
 
-The Distelli module currently installs, configures, and manages the Distelli Agent service across \*nix-centric and Windows based operating systems.
+1. [Usage](#usage)
+1. [Reference](#reference)
+1. [Contributing](#contributing)
 
 ## Usage
 
-All parameters for the Distelli module are contained within the main `::distelli::agent` class, so for any function of the module, set the options you want. See the common usages below for examples.
+### TL;DR
```

Comment:
Prefer not to use this kind of acronym.  Not every knows what this is.  Keep the "Begging with..."